server/Makefile

```diff
@@ -89,6 +91,16 @@ else
 	BUILD_TYPE_NAME = team
 endif
 
+# Clean up the old means of importing enterprise source, if it exists
+ifneq ($(wildcard channels/imports/imports.go),)
+	IGNORE := $(shell rm -f channels/imports/imports.go)
```

Comment:
Any objection to this, @agnivade & @crspeller? I literally ran into the thing I was worried about today, scratched my head, then realized my mistake. I'd prefer not to have this repeated over and over again as people upgrade, or later as they switch between branches.

Reply:
No objection. I would prefer not to have to remember to delete this file constantly.

Reply:
Fine by me!